packages/components/src/Chip/Chip.css

```
@@ -23,6 +22,15 @@
   transition: all var(--ease-both) var(--timing-quick);
   align-items: center;
 }
+div.chip {
```

As Darryl would say - let them breathe 😆 

```suggestion

div.chip {
```

---

Done!